pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -174,3 +176,103 @@ func aggregateJobStatus(object *unstructured.Unstructured, aggregatedStatusItems
 	job.Status = *newStatus
 	return helper.ToUnstructured(job)
 }
+
+func aggregateDaemonSetStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	daemonSet, err := helper.ConvertToDaemonSet(object)
+	if err != nil {
+		return nil, err
+	}
+
+	oldStatus := &daemonSet.Status
+	newStatus := &appsv1.DaemonSetStatus{}
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+		temp := &appsv1.DaemonSetStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+		klog.V(3).Infof("Grab daemonSet(%s/%s) status from cluster(%s), currentNumberScheduled: %d, desiredNumberScheduled: %d, numberAvailable: %d, numberMisscheduled: %d, numberReady: %d, updatedNumberScheduled: %d",
+			daemonSet.Namespace, daemonSet.Name, item.ClusterName, temp.CurrentNumberScheduled, temp.DesiredNumberScheduled, temp.NumberAvailable, temp.NumberMisscheduled, temp.NumberReady, temp.UpdatedNumberScheduled)
+		newStatus.ObservedGeneration = daemonSet.Generation
+		newStatus.CurrentNumberScheduled += temp.CurrentNumberScheduled
+		newStatus.DesiredNumberScheduled += temp.DesiredNumberScheduled
+		newStatus.NumberAvailable += temp.NumberAvailable
+		newStatus.NumberMisscheduled += temp.NumberMisscheduled
+		newStatus.NumberReady += temp.NumberReady
+		newStatus.UpdatedNumberScheduled += temp.UpdatedNumberScheduled
```

Comment:
Just a question: why not consider `NumberUnavailable`?

Reply:
I test and it seems `NumberUnavailable` not in `aggregatedStatus`.
![image](https://user-images.githubusercontent.com/26862112/162113936-0508556b-85e2-4150-b658-35ceedd6e540.png)

Reply:
The daemonset status in the member cluster also doesn't have this field value?

Reply:
yeah use `kubectl get ds/nginx -oyaml`. Also no this field value.

Reply:
It seems ds controller use desiredNumberScheduled - numberAvailable to explain numberUnavailable

Reply:
any link to the relevant code?

Reply:
see [here](https://github.com/kubernetes/kubernetes/blob/707b8b6efd1691b84095c9f995f2c259244e276c/pkg/controller/daemon/daemon_controller.go#:~:text=numberUnavailable%20%3A%3D%20desiredNumberScheduled%20%2D%20numberAvailable)

Reply:
> yeah use `kubectl get ds/nginx -oyaml`. Also no this field value.

The `NumberUnavailable` is omitempty, if NumberUnavailable is zero, it won't present.

Reply:
> It seems ds controller use desiredNumberScheduled - numberAvailable to explain numberUnavailable

![image](https://user-images.githubusercontent.com/74236054/162356500-bfdeff2b-897e-4928-9ca2-80150287f3bb.png)

Reply:
you are right. numberMisscheduled is not omit but numberunavailable is.Thanks